sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -249,7 +249,11 @@ public RecordedData getRecordedData() {
     public Supplier<String> getProxyVariableSupplier() {
         return () -> {
             Objects.requireNonNull(this.testProxyPlaybackClient, "Playback must be started to retrieve values");
-            return proxyVariableQueue.remove();
```

Comment:
<img width="491" alt="image" src="https://user-images.githubusercontent.com/16845631/221705118-ed237536-c4e2-43c8-86a3-97d60a5126b0.png">

This happens when a recording for that test is not found but the playback client is successfully registered.